app/controllers/simplified-account/settings/index.controller.js

```diff
@@ -0,0 +1,20 @@
+const formatSimplifiedAccountPathsFor = require('../../../utils/simplified-account/format/format-simplified-account-paths-for')
+const paths = require('../../../paths')
+const { LIVE } = require('../../../models/go-live-stage')
+
+function get (req, res) {
+  const account = req.account
+  const service = req.service
+  // the default setting for the index view is dependent on the account type and go live state
+  if (account.type === 'test' && service.currentGoLiveStage !== LIVE) {
+    return res.redirect(formatSimplifiedAccountPathsFor(paths.simplifiedAccount.settings.serviceName.index, req.account.service_id, req.account.type))
+  } else if (account.type === 'live') {
+    return res.redirect(formatSimplifiedAccountPathsFor(paths.simplifiedAccount.settings.serviceName.index, req.account.service_id, req.account.type))
+  } else {
+    return res.redirect(formatSimplifiedAccountPathsFor(paths.simplifiedAccount.settings.emailNotifications.index, req.account.service_id, req.account.type))
```

Comment:
Either the `else`s or the `return`s can be removed here, both together is unnecessary.

Option a:

```
if (condition1) {
  return doAction1()
}
if (condition2) {
  return doAction2()
}
doDefaultAction()
```

Option b:

```
if (condition1) {
  doAction1()
} else if (condition2) {
  doAction2()
} else {
  doDefaultAction()
}
```

They are functionally equivalent so I think it's a case of chosing which one you prefer.